src/dev/konnector.md

```diff
@@ -508,6 +508,33 @@ Cozy will soon have a store for connectors and you will be able to publish conne
 at the moment, you need to declare your new connector on the [cozy forum](https://forum.cozy.io).
 The Cozy team will review your code and add your connector to the [Cozy Collect] application.
 
+To make the connector available more quickly for all cozys, you can follow this few steps of 
+pre-packaging:
+
+### Icon
+
+You need to push an icon in `assets/icon.png`. Please respect this rules :
+ - Square icon, possibly a png or svg
+ - Try the apple store icon if needed
```

Comment:
In fact, the applications store for iOS (and macOS) is called _App Store_. _Apple Store_ is a real store where Apple sells its macbook and other devices.
> Try to find an icon on the App Store (they use square icons)